config/stamp/default/parameters.json

```diff
@@ -0,0 +1,272 @@
+        },
+        "mongoVmSize": {
+            "value": "Standard_D2_v2"
+        },
+        "mysqlVmSize": {
+            "value": "Standard_D2_v2"
+        },
+        "frontendVmSize": {
+            "value": "Standard_D2_v2"
+        },
+        "adminUsername": {
+            "value": "lexoxaadmin"
+        },
+        "adminPublicKey": {
+            "value": "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQCj0GHnhX8L8cPtCFhNPTClvDMzeMsB+ys5uMaqFgTa2mrWSUBsbsVwK91H6mwtrKRJf42Ry5DVYKLPf8XSz22v5iyZ1DrbkGJcmHRWEBFA47habQFg+eXBUSENUf1cMOh4RwkN1olCLpsdeKrF0Rj6k3ZFODUJIf+v5HYKFkj+8iVdQh0wgnPVxneqIKcbqNLO8ymmAiTT+fYaubPWzJmq8nUKoNZRiPXSSAwHHtNouR4vvPtOU7je2TxzoYIcM/Ja6YycBx44W+A7ygY5/hmij4Byhm20MLZLoBLX2AMp8horzqWhdF/wvFMhv8FTTzlWpwhuEpbd2M4B6Tylajm8lwA9u/9MO6lEbqnX/XH/KS9hPZdKvqXgbPTzM265Aea0LPXHDdF0f9AID6Xl3grHrtkKFGMzbgqFtF9nd51kwDTZr2b1zIVWb6XxDXXVhX4XqWniCT8xEzuGyMp4/mhaghRDhT45Alu6Frv6d5+IjeabBSXOwwe+8NpTDc0+hxM/MYiE4K6EHpPqwxOoRJcZFrjdSxQGGrCD+bLJbGfI8tn96Wq4zBBUMp+iX1iXXNScgqUAaSoNKEH8ag177oOvtH3AsB/b7Nm/eUIr/WYfYESlft1M1h25Lvu6QgFyqJlwdXSPCiIYbR6nK6WI2Zz6cActCLoJaN7IPw6RLeY+tw== admin@contoso.com"
+        },
+        "mongoServerAdminUserName": {
+            "value": "oxamongoadmin"
+        },
+        "mongoServerAdminPassword": {
+            "value": "{SERVICEACCOUNTPASSWORD}"
+        },
+        "mysqlServerPackageVersion": {
+            "value": "5.6"
+        },
+        "mysqlServerAdminUserName": {
+            "value": "oxamysqladmin"
+        },
+        "mysqlServerAdminPassword": {
+            "value": "{SERVICEACCOUNTPASSWORD}"
+        },
+        "mysqlServerReplUserName": {
+            "value": "oxamysqlrepluser"
+        },
+        "mysqlServerReplPassword": {
+            "value": "{SERVICEACCOUNTPASSWORD}"
+        },
+        "mysqlBackupUserName": {
+            "value": "oxamysqlbackupuser"
+        },
+        "mysqlBackupUserPassword": {
+            "value": "{SERVICEACCOUNTPASSWORD}"
+        },
+        "deploymentSlot": {
+            "value": "slot1"
+        },
+        "deploymentVersionId": {
+            "value": "201703141409"
+        },
+        "autoScaleEnabled": {
+            "value": true
+        },
+        "autoScaleCapacityDefault": {
+            "value": 1
+        },
+        "autoScaleCapacityMaximum": {
+            "value": 1
+        },
+        "autoScaleCapacityMinimum": {
+            "value": 1
+        },
+        "oxaToolsGithubAccountName": {
+            "value": "Microsoft"
+        },
+        "oxaToolsGithubProjectName": {
+            "value": "oxa-tools"
+        },
+        "oxaToolsGithubBranch": {
+            "value": "oxa/master.fic.eltonc"
+        },
+        "edxConfigurationGithubAccountName": {
+            "value": "Microsoft"
+        },
+        "edxConfigurationGithubProjectName": {
+            "value": "edx-configuration"
+        },
+        "edxConfigurationGithubBranch": {
+            "value": "oxa/master.fic.eltonc"
```

Comment:
I think Elton took my advice :) We have a reference to Elton's branch. Should this be oxa/master.fic?